packages/core/README.md

```diff
@@ -135,6 +135,7 @@ export default createPrompt((config, done) => {
   const windowedChoices = usePagination(allChoices, {
     active: cursorPosition,
     pageSize: config.pageSize,
+    loop: config.loop,
```

Comment:
Making a note for later, we'll need to document `render` & `items` here.

Reply:
I've made some changes to the README, please take a look.